File: baekjoon/python/ex11021.py
import sys


if __name__ == "__main__":
    n = int(sys.stdin.readline().strip())
    for _ in range(n):
        print(
            "Case #{}: {}"
            .format(
                _ + 1,
                sum(list(map(int, sys.stdin.readline().split())))
            )
        )

File: baekjoon/nodejs/src/ex11021.js
const solution = (input) =>
    input
        .slice(1, input[0] + 1)
        .map((s) => s.split(' ').reduce((acc, n) => acc + +n, 0))
        .map((n, idx) => `Case #${idx + 1}: ${n}`)
        .join('\n');

const input = [];
require('readline')
    .createInterface({ input: process.stdin })
    .on('line', (line) => input.push(line))
    .on('close', (_) => {
        console.log(solution(input));
        process.exit(0);
    });

// test code
test('example1', () => {
    const input = `5
1 1
2 3
3 4
9 8
5 2`.split('\n');
    const output = `Case #1: 2
Case #2: 5
Case #3: 7
Case #4: 17
Case #5: 7`;
    expect(solution(input)).toEqual(output);
});
